tests/cephadm/test_cephadm_upgrade.py

```diff
@@ -3,10 +3,88 @@
 
 """
 import logging
+from concurrent.futures import ThreadPoolExecutor
+from time import time
 
 from ceph.ceph_admin.orch import Orch
 
-LOG = logging.getLogger()
+LOG = logging.getLogger(__name__)
+
+
+class RadosBench:
+    def __init__(self, client, pool=""):
+        """
+        Initialize RadosBench
+        """
+        self.client = client
+        self.pool = pool or self.create_pool()
+
+    @staticmethod
+    def create_run_name():
+        """
+        Return run name
+        """
+        return f"test-run-{int(time())}"
+
+    def create_pool(self):
+        """
+        Create pool and enable application
+        """
+        name = f"pool-{int(time())}"
+        self.client.exec_command(cmd=f"ceph osd pool create {name} 128 128", sudo=True)
```

Comment:
we can use PG values and pool names as well as parameters, also with a default value of course.

Reply:
In 5.x, we don't need to execute this with root level privileges... may we could add an attribute holding the RHCS version.

Reply:
> we can use PG values and pool names as well as parameters, also with a default value of course.

yes, agree. will take care when this is added into library.